corehq/elastic.py

```diff
@@ -9,6 +9,7 @@
 
 from memoized import memoized
 
+from corehq.util.datadog.gauges import datadog_counter
```

Comment:
curious why this one wasn't changed?
```suggestion
from corehq.util.metrics import metrics_counter
```
Will also need to change
https://github.com/dimagi/commcare-hq/blob/69fb53ba2ebf60056ad4bffd431a8aabd4d3531b/corehq/elastic.py#L508

Reply:
I wanted to get these other changes out first